index.html

```diff
@@ -0,0 +1,1117 @@
+        </li>
+        <li class="nav-item">
+          <a href="#" class="nav-link border-right">Newsgeek</a>
+        </li>
+        <li class="nav-item">
+          <a href="#" class="nav-link border-right">Sports</a>
+        </li>
+        <li class="nav-item">
+          <a href="#" class="nav-link border-right">Health</a>
+        </li>
+        <li class="nav-item">
+          <a href="#" class="nav-link border-right">The Debate</a>
+        </li>
+        <li class="nav-item">
+          <a href="#" class="nav-link border-right">Vantage</a>
+        </li>
+        <li class="nav-item">
+          <a href="#" class="nav-link">Weather</a>
+        </li>
+      </ul>
+    </div>
+    <input type="text" placeholder="Search" class="col-lg-1 search-input mt-lg-2">
+    <div class="col-lg- trapezium"></div>
+  </header>
+  <main class="container-fluid mt-md-4 p-0">
+    <div class="row">
+      <div class="d-flex col-sm-12 col-lg-8 feat order-2 order-md-0">
+        <section id="feature-stories" class="col-sm-12 col-md-5 col-lg-4 min-content-height sticky-top p-0">
+          <h2>FEATURED STORIES</h2>
+          <div>
+            <img
+              src="https://d.newsweek.com/en/full/1575707/coronavirus.jpg?w=397&h=265&q=90&f=9d411f587ddecef5bee8f0f99ca54eaf"
+              alt="" class="w-100">
+            <h3 class="h3-feature-story">COVID-19 Pandemic Could Cost U.S. $7 Trillion, Professor Estimates</h3>
+            <p class="p-feature-story">"Without vigorous steps, the CDC estimates the COVID-19 will infect 70 to 150
+              million Americans,"
+              Michael
+              Hicks tells Newsweek. "This would have enormous economic costs."</p>
+          </div>
+          <div>
+            <img
+              src="https://d.newsweek.com/en/full/1575699/ohio-coronavirus.jpg?w=397&h=265&q=90&f=899960ae2e480fbe4b567b095d603d01"
+              alt="" class="w-100">
+            <h3 class="h3-feature-story">Ohio and Louisiana Issue Statewide Stay-at-Home Orders Amid Coronavirus</h3>
+            <p class="p-feature-story">Ohio's order mandating all residents stay at home will be implemented Monday
+              evening at 11.59 p.m.
+              local
+              time, and is scheduled to end on April 6.</p>
+          </div>
+          <div>
+            <img
+              src="https://d.newsweek.com/en/full/1575668/rand-paul.jpg?w=397&h=265&q=90&f=6f09981a0ba49e99cd98b49fdd5ff2c8"
+              alt="" class="w-100">
+            <h3 class="h3-feature-story">Rand Paul Tests Positive for Coronavirus</h3>
+            <p class="p-feature-story">He expects to be back in the Senate after his quarantine period ends," according
+              to the Kentucky
+              senator's
+              Twitter account.</p>
+          </div>
+        </section>
+        <section id="top-stories" class="d-none d-md-block d-lg-block col-md-7 col-lg-8">
+          <h2>top Story</h2>
+          <div>
```

Comment:
The images on this section and the Featured stories should have a white badge with red text on the bottom-left corner. Please add them.